official/README.md

```diff
@@ -2,34 +2,64 @@
 
 If you are on a version of TensorFlow earlier than 1.4, please [update your installation](https://www.tensorflow.org/install/).
 
----
+## Requirements
+Please follow the below steps before running models in this repo:
+
+1. Add the top-level ***/models*** folder to the Python path with the command:
+   ```
+   export PYTHONPATH="$PYTHONPATH:/path/to/models"
+   ```
+2. Install dependencies:
+   ```
+   pip3 install --user -r official/requirements.txt
+   ```
+   or
+   ```
+   pip install --user -r official/requirements.txt
+   ```
 
-Below is a list of the models available.
 
-[boosted_trees](boosted_trees): A Gradient Boosted Trees model to classify higgs boson process from HIGGS Data Set.
+To make Official Models easier to use, we are planning to create a pip installable Official Models package. This is being tracked in [#917](https://github.com/tensorflow/models/issues/917).
 
-[mnist](mnist): A basic model to classify digits from the MNIST dataset.
 
-[resnet](resnet): A deep residual network that can be used to classify both CIFAR-10 and ImageNet's dataset of 1000 classes.
+## Available models
 
-[wide_deep](wide_deep): A model that combines a wide model and deep network to classify census income data.
+**NOTE:** Please make sure to follow the steps in the [Requirements](#requirements) section.
 
-More models to come!
+* [boosted_trees](boosted_trees): A Gradient Boosted Trees model to classify higgs boson process from HIGGS Data Set.
+* [mnist](mnist): A basic model to classify digits from the MNIST dataset.
+* [resnet](resnet): A deep residual network that can be used to classify both CIFAR-10 and ImageNet's dataset of 1000 classes.
+* [transformer](transformer): A transformer model to translate the WMT English to German dataset.
+* [wide_deep](wide_deep): A model that combines a wide model and deep network to classify census income data.
+* More models to come!
 
 If you would like to make any fixes or improvements to the models, please [submit a pull request](https://github.com/tensorflow/models/compare).
 
----
+## New Models
 
-## Running the models
+The team is actively working to add new models to the repository. Every model should follow the following guidelines, to uphold the
+our objectives of readable, usable, and maintainable code.
 
-The *Official Models* are made available as a Python module. To run the models and associated scripts, add the top-level ***/models*** folder to the Python path with the command: `export PYTHONPATH="$PYTHONPATH:/path/to/models"`
+**General guidelines**
+* Code should be well documented and tested.
+* Runnable from a blank environment with relative ease.
+* Trainable on: single GPU/CPU (baseline), multiple GPUs, TPU
+* Compatible with Python 2 and 3 (using [six](https://pythonhosted.org/six/) when necessary)
```

Comment:
Maybe also: `* Conform to [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)`